docs/source/installation.rst

```diff
@@ -2,152 +2,125 @@ Installation
-++++++++++++++++++++++++++++++++++++++
 
-::
+    .. tab:: Compute Canada HPC
 
-    pip install -e .
+        There are numerous constraints and limited package availabilities with ComputeCanada cluster environment.
 
-Approach 2: Venv
------------------
+        It is best to attempt ``venv`` based installations and follow up with ComputeCanada technicall support as MANY specially compiled packages (e.g. numpy) are exclusively available for Compute Canada HPC environment.
 
-Step 1: Setup Python Virtual Environment.
-+++++++++++++++++++++++++++++++++++++++++
+        If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_ .
 
-``ivadomed`` requires Python >= 3.6 and <3.9 (If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules (e.g. python 3.9) as mentioned  `here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_ ). We recommend
-working under a virtual environment, which could be set as follows:
 
-::
+Step 2: Install ``ivadomed``
+++++++++++++++++++++++++++++
 
-    virtualenv venv-ivadomed
-    source venv-ivadomed/bin/activate
 
+.. tabs::
 
-.. warning::
-   If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
+    .. tab:: Pypi Installation
 
-   ::
+        Install ``ivadomed`` and its requirements from
+        `PyPI <https://pypi.org/project/ivadomed/>`__:
 
-     virtualenv venv-ivadomed --python=python3.6
+        ::
 
-Step 2: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            pip install --upgrade pip
 
+            pip install ivadomed
 
-::
+    .. tab:: Repo Installation (Advanced or Developer)
 
-    git clone https://github.com/ivadomed/ivadomed.git
-    cd ivadomed
- 
+        Bleeding-edge developments are available on the project's master branch
+        on Github. Installation ``ivadomed`` from source:
 
-Step 3: Install PyTorch 1.5 and TorchVision
-+++++++++++++++++++++++++++++++++++++++++++
+        ::
 
-If you have a compatible NVIDIA GPU that supports CUDA, run the following command:
+            git clone https://github.com/ivadomed/ivadomed.git
 
-::
-   
-   pip install -r requirements_gpu.txt
+            cd ivadomed
 
+            pip install -e .
 
-If you do not have a compatible GPU, run the following installer to use ``ivadomed`` with CPU.
 
+Step 3: Install ``ivadomed`` with CPU or GPU Support, Install ``torch`` and ``torchvision``
```

Comment:
```suggestion
Step 3: Install ``torch`` and ``torchvision`` with CPU or GPU Support. 
```

I'm not sure about this, but if I understand correctly "ivadomed" is already instaled at step 2 and step 3 is only for torch and torchvision. Feel free to disregard if I'm mistaken.
The length of the title underline below also has to be adjusted accordingly.

Reply:
Very good point. I agree the title could be more clear. Changed. :D